test/integ/pandas/test_pandas_tools.py

```diff
@@ -139,9 +139,10 @@ def test_write_pandas_with_overwrite(
         "gzip",
     ],
 )
-@pytest.mark.parametrize("quote_identifiers", [True, False])
-@pytest.mark.parametrize("auto_create_table", [True, False])
-@pytest.mark.parametrize("create_temp_table", [True, False])
+@pytest.mark.parametrize("quote_identifiers", [True])
+@pytest.mark.parametrize("auto_create_table", [True])
+@pytest.mark.parametrize("create_temp_table", [True])
```

Comment:
Did you mean to turn the 8 tests into 1?

Reply:
We should remove `create_temp_table` and replace `if create_temp_table` with `if table_type.lower() in ("temp", "temporary")`

Reply:
Crap this is not intended. I changed this to quickly verify some changes 😂 Will revert

Reply:
BTW I still want to test `create_temp_table` because I have a deprecation warning check. Both `table_type` and `create_temp_table` will be passed to `write_pandas` and we test their expected behavior respectively.